graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -1,18 +1,35 @@
-import itertools
+import datetime
+import git
+import os
 import random
+import sys
 
-from .animals import get_animal_generators
-from .species import get_species_generators
+from .animals import get_animal_generation_commands
+from .species import get_species_generation_commands
 
 
-random.seed(0)
+def main():
+    random.seed(0)
 
+    module_path = os.path.relpath(__file__)
```

Comment:
`relpath` is relative to the current working directory, which can easily change as the user is using their shell. Let's either use `abspath` or use just the module name `generate_test_sql`.